modin/pandas/series.py

```diff
@@ -484,7 +485,9 @@ def values(self):  # noqa: RT01, D200
         """
         Return Series as ndarray or ndarray-like depending on the dtype.
         """
-        return self.to_numpy()
+        if not is_numeric_dtype(self.dtype):
+            return self._default_to_pandas("values")
```

Comment:
what are these non-numeric dtypes whose behavior deviates from a common `.to_numpy()`? If it's only categories can we then just apply the same approach as for `.ravel()`?:
https://github.com/modin-project/modin/blob/3e314d8686293415450e5d382547262f49aa38d8/modin/pandas/series.py#L1502-L1505

Reply:
Code snippet from pandas:
```python
        Overview:

        dtype       | values        | _values       | array         |
        ----------- | ------------- | ------------- | ------------- |
        Numeric     | ndarray       | ndarray       | PandasArray   |
        Category    | Categorical   | Categorical   | Categorical   |
        dt64[ns]    | ndarray[M8ns] | DatetimeArray | DatetimeArray |
        dt64[ns tz] | ndarray[M8ns] | DatetimeArray | DatetimeArray |
        td64[ns]    | ndarray[m8ns] | TimedeltaArray| ndarray[m8ns] |
        Period      | ndarray[obj]  | PeriodArray   | PeriodArray   |
        Nullable    | EA            | EA            | EA            |
```

There seems to be a case with the `Nullable` type where EA (extended array?) would be returned.

Reply:
EA = ExtensionArray